Week1/homework/world-queries.txt

```diff
@@ -1,40 +1,231 @@
-Answers for the world queries go here:
-
-For example:
+﻿Answers for the world queries go here:
 
 1. What are the names of all the cities in the Netherlands?
 
 SELECT city.Name FROM city JOIN country ON city.CountryCode = country.Code WHERE country.Name = 'Netherlands';
-+-------------------+
-| Name              |
-+-------------------+
-| Amsterdam         |
-| Rotterdam         |
-| Haag              |
-| Utrecht           |
-| Eindhoven         |
-| Tilburg           |
-| Groningen         |
-| Breda             |
-| Apeldoorn         |
-| Nijmegen          |
-| Enschede          |
-| Haarlem           |
-| Almere            |
-| Arnhem            |
-| Zaanstad          |
-| ´s-Hertogenbosch  |
-| Amersfoort        |
-| Maastricht        |
-| Dordrecht         |
-| Leiden            |
-| Haarlemmermeer    |
-| Zoetermeer        |
-| Emmen             |
-| Zwolle            |
-| Ede               |
-| Delft             |
-| Heerlen           |
-| Alkmaar           |
-+-------------------+
-28 rows in set (0.00 sec)
\ No newline at end of file
+ +-----------------------+
+ | Name                  |
+ +-----------------------+
+ | Amsterdam             |
+ | Rotterdam             |            
+ | Haag                  |     
+ | Utrecht               |     
+ | Eindhoven             | 
+ | Tilburg               |     
+ | Groningen             | 
+ | Breda                 |    
+ | Apeldoorn             | 
+ | Nijmegen              | 
+ | Enschede              | 
+ | Haarlem               |  
+ | Almere                |   
+ | Arnhem                | 
+ | Zaanstad              | 
+ | ´s-Hertogenbosch      |
+ | Amersfoort            |
+ | Maastricht            |
+ | Dordrecht             |
+ | Leiden                |  
+ | Haarlemmermeer        |
+ | Zoetermeer            |
+ | Emmen                 |
+ | Zwolle                |   
+ | Ede                   |    
+ | Delft                 |    
+ | Heerlen               | 
+ | Alkmaar               |
+ +------------------------
+28 rows in set (0.00 sec)
+---------------------------------------------------------------------------------------------------
+
+
+2.What is the population of Rotterdam ?
+
+SELECT Population FROM city WHERE Name = 'Rotterdam';
+
+ +-----------------------+
+ | Population            |
+ +-----------------------+
+ | 593321                |
+ +-----------------------+
+
+1 row in set (0,01 sec)
+---------------------------------------------------------------------------------------------------
+
+3.What's the name of all the countries on the continent ‘North America’ ?
+
+SELECT DISTINCT Name FROM country WHERE Continent = 'North America' ;
```

Comment:
:+1: 

You might not have needed 'DISTINCT' for countries, but for cities it can be very useful.